src/main/java/peoplesoft/model/job/Money.java

```diff
@@ -37,6 +39,15 @@ public Money(BigDecimal value) {
         this.value = value.setScale(VALUE_SCALE, RoundingMode.HALF_UP);
     }
 
+    /**
+     * Throws NegativeMoneyException if {@code money} is negative.
+     */
+    public static void requireNonNegative(Money money) {
```

Comment:
would it be better to have it in a separate util class, eg `MoneyUtil`? i think some of the existing model classes do this

Reply:
I have added a equivalent class for Duration as well. For now, the Util classes can be found under model > job. I'm not sure if there should be a common Util folder for all models instead.